site/content/en/docs/commands/start.md

```diff
@@ -77,6 +77,7 @@ minikube start [flags]
       --namespace string                  The named space to activate after start (default "default")
       --nat-nic-type string               NIC Type used for nat network. One of Am79C970A, Am79C973, 82540EM, 82543GC, 82545EM, or virtio (virtualbox driver only) (default "virtio")
       --native-ssh                        Use native Golang SSH client (default true). Set to 'false' to use the command line 'ssh' command when accessing the docker machine. Useful for the machine drivers when they will not start with 'Waiting for SSH'. (default true)
+      --network string                    Docker network to run minikube with. Only available with the docker driver. If left empty, minikube will create a new network.
```

Comment:
please add Podman here, we added podman in last version of minikube.

Reply:
sure, done